src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -414,7 +414,7 @@ public String getDataInstanceDir() {
   // ============== resources begin ======================================
 
   /**
-   * Non-static getResourceDir() will return /resource/{ServiceType}
+   * Non-static getResourceDir() will return /resource/{MetaData}

```

Comment:
This seems wrong.. the resource directory is actually based off the string identifier of the service type.

Reply:
Your correct - overzealous search and replace ... corrected with example